hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -11,12 +11,19 @@ parameters:
 - name: REPO_NAME
   value: managed-velero-operator
   required: true
+- name: OPERATOR_NAME
+  value: managed-velero-operator
```

Comment:
do we need to duplicate this? couldn't we just use REPO_NAME?

Reply:
We can use `REPO_NAME`, no problem. It's just not every operator template has this, so that's why I came up with the `REPO_NAME`. I propose to use the `REPO_NAME` for those who have this, and introduce this to those who don't. WDYT?